scheduler/src/modules/transactions/transaction-detail.entity.ts

```diff
@@ -0,0 +1,25 @@
+import { Entity, Column, PrimaryGeneratedColumn, OneToOne, JoinColumn } from 'typeorm';
+import { Transaction } from './transaction.entity';
+
+@Entity('ORBC_CFS_TRANSACTION_DETAIL', { schema: 'permit' })
+export class TransactionDetail {
```

Comment:
Would recommend to prefix the Class name with CFS in order to avoid any confusion with ORBC_TRANSACTION and related tables